config/config.ts

```diff
@@ -19,6 +19,51 @@ import {
   DISCORD_PROFILE_SERVICE_STAGING_HELP_GROUP,
   DISCORD_PROFILE_SERVICE_DEVELOPMENT_HELP_GROUP,
 } from "../src/constants/variables";
+import { config as configFromDotEnv } from "dotenv";
+
+export function loadEnv(env: env, fromWorkerEnv: boolean): env {
+  const Env: env = {
+    //if `fromWokerEnv` is true, then load from the `env` passed as argument to the function,
+    // else if `fromWokerEnv` is false, load from process.env
+    //(or set to '' if value from process.env is undefined) to avoid Error TS2322
+    CURRENT_ENVIRONMENT: fromWorkerEnv
+      ? env.CURRENT_ENVIRONMENT
+      : process.env.CURRENT_ENVIRONMENT || "",
+    DISCORD_APPLICATION_ID: fromWorkerEnv
+      ? env.DISCORD_APPLICATION_ID
+      : process.env.DISCORD_APPLICATION_ID || "",
+    DISCORD_GUILD_ID: fromWorkerEnv
+      ? env.DISCORD_GUILD_ID
+      : process.env.DISCORD_GUILD_ID || "",
+    DISCORD_TOKEN: fromWorkerEnv
+      ? env.DISCORD_TOKEN
+      : process.env.DISCORD_TOKEN || "",
+    DISCORD_PUBLIC_KEY: fromWorkerEnv
+      ? env.DISCORD_PUBLIC_KEY
+      : process.env.DISCORD_PUBLIC_KEY || "",
+    CLOUDFLARE_API_TOKEN: fromWorkerEnv
+      ? env.CLOUDFLARE_API_TOKEN
+      : process.env.CLOUDFLARE_API_TOKEN || "",
+    CLOUDFLARE_ACCOUNT_ID: fromWorkerEnv
+      ? env.CLOUDFLARE_ACCOUNT_ID
+      : process.env.CLOUDFLARE_ACCOUNT_ID || "",
+    BOT_PRIVATE_KEY: fromWorkerEnv
+      ? env.BOT_PRIVATE_KEY
+      : process.env.BOT_PRIVATE_KEY || "",
+    RDS_SERVERLESS_PUBLIC_KEY: fromWorkerEnv
+      ? env.RDS_SERVERLESS_PUBLIC_KEY
+      : process.env.RDS_SERVERLESS_PUBLIC_KEY || "",
+    CRON_JOBS_PUBLIC_KEY: fromWorkerEnv
+      ? env.CRON_JOBS_PUBLIC_KEY
+      : process.env.CRON_JOBS_PUBLIC_KEY || "",
+    IDENTITY_SERVICE_PUBLIC_KEY: fromWorkerEnv
+      ? env.IDENTITY_SERVICE_PUBLIC_KEY
+      : process.env.IDENTITY_SERVICE_PUBLIC_KEY || "",
```

Comment:
Why set it to '' empty string?

Reply:
Also, what are 2 things here, `env.CURRENT_ENVIRONMENT` and `process.env.CURRENT_ENVIRONMENT`?
Won't env always come from process.env?

Reply:
env is the object that is present only in the worker, it contains the env vars injected as secrets into the worker after [deployment](https://github.com/Saitharun279/discord-slash-commands/blob/e29109b7fd5270e12741bd48b93cc2f6f8833c5e/.github/workflows/register-commands-staging.yaml#L55)
they are not available in the worker from process.env. 

But when CD pipeline is running, they are accessed from the process.env, which are injected as env from Github Actions secrets